app/logic/Calendar/EWS/EWSEvent.ts

```diff
@@ -400,8 +400,28 @@ export class EWSEvent extends Event {
 
   async respondToInvitation(response: InvitationResponseInMessage): Promise<void> {
     assert(this.isIncomingMeeting, "Only invitations can be responded to");
+    let itemID = this.itemID;
+    // Unfortunately this API won't take an OccurrenceItemId directly.
+    if (!itemID) {
+      assert(this.recurrenceCase == RecurrenceCase.Instance, "must be an instance");
```

Comment:
```suggestion
      // In case the invitation is for a single instance of a recurring meeting
      assert(this.recurrenceCase == RecurrenceCase.Instance, "must be an instance");
```